docs/source/concept_models/vivarium_census_synthdata/concept_model.rst

```diff
@@ -486,6 +511,10 @@ simulant is initialized, and to come up with a consistent value for
 the "reference person" relationship field, we use the following
 mapping:
 
+In addition, tracked mothers will be assigned as "guardians" based on the 
```

Comment:
I think you'll want to move this above the previous paragraph, so it doesn't go between `the following mapping:` and the mapping.

Reply:
Thanks!